code/modules/client/asset_cache.dm

```diff
@@ -325,6 +325,13 @@ GLOBAL_LIST_EMPTY(asset_datums)
 	var/size_id = sprite[SPR_SIZE]
 	return {"<span class="[name][size_id] [sprite_name]"></span>"}

```

Comment:
should use the new helper here for consistency.

Reply:
You mean we can't leave `icon_class_name` helper to be used for future interfaces? Just saying, Inferno can't render html tags like that.